docs/solvation.ipynb

```diff
@@ -46,10 +56,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
    "id": "d40fed9d-5888-4bee-b014-0d6430eea8c0",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "numpy: 1.26.3\n",
+      "scipp: 23.08.0\n",
+      "EasyReflectometry: 0.0.2\n",
+      "refnx: 0.1.41\n"
+     ]
+    }
+   ],
```

Comment:
all these extra outputs mean that you probably didn't clean/reset the notebook before committing. This should be done so the initial state is an empty notebook.

Reply:
No, they are intended.  It is a clean notebook.  The reason for the new "code" parts is that I added a new markdown section stating that we setup the matplotlib to plot the figures inline and import the needed modules:

```
%matplotlib inline

import ...
```

Reply:
Had misunderstood PR comment:

Removed output form solvation.ipynb